django/thunderstore/core/utils.py

```diff
@@ -65,33 +62,3 @@ def capture_exception(exception: Exception) -> None:
         raise exception
     else:
         capture_sentry_exception(exception)
-
-
-def _create_ghost_user_username(id_: str) -> str:
-    return f"Ghost User {id_}"
-
-
-def _create_ghost_user_email(id_: str) -> str:
-    # The ID used in ghost user usernames and emails are not the same as their `User.id`
-    # This is because `User` still uses an autoincrement ID
-    return f"{id_}.gu@thunderstore.io"
-
-
-@transaction.atomic
-def delete_user(user: User) -> None:
```

Comment:
Removing the util function and requiring the celery task to be called directly might be annoying although it is only currently called in 2 places, one of which being a test. I tried importing the task into utils but got a circular import.

Reply:
Calling the celery task shouldn't be too different form calling the utility function, or am I missing something here?